### PR TITLE
Removed HME0, HME1, HME2 config

### PR DESCRIPTION
**Pull request by dseah-intel**
- Remove the ability to set HME0, HME1, HME2 and its width/height

### CODE REVIEW

**Comment by hassount** [comment]
@dseah-intel 
Did you test bit exactness for this PR ?

**Comment by dseah-intel** [comment]
@hassount The change was tested on the original branch before the merge with master

**Comment by tianjunwork** [comment]
Please update the user guide accordingly. the doc version is uploaded to otc branch.

**Comment by tianjunwork** [comment]
Hi @dseah-intel could you update user guide from https://github.com/otcshare/SVT-HEVC/tree/master/Docs and upload the pdf? #69 is dependant on your change. Thx.

**Comment by tianjunwork** [comment]
And does SVT_FunctionalTests.py also need to be updated?

**Comment by dseah-intel** [comment]
Hi Jun,

I will update the test script and user doc as soon as I can.

Cheers,
Daryl

**Comment by tianjunwork** [comment]
@dseah-intel Thank you:)

File: Source/Lib/Codec/EbMotionEstimationProcess.c [hunk 1]
```diff
@@ -299,24 +298,6 @@ void* SetMeHmeParamsFromConfig(
     meContextPtr->searchAreaWidth = (EB_U8)sequenceControlSetPtr->staticConfig.searchAreaWidth;
     meContextPtr->searchAreaHeight = (EB_U8)sequenceControlSetPtr->staticConfig.searchAreaHeight;
 
-    meContextPtr->numberHmeSearchRegionInWidth = (EB_U16)sequenceControlSetPtr->staticConfig.numberHmeSearchRegionInWidth;
```
[the remark this answers is not in the document]
Comment (dseah-intel) [reply Source/Lib/Codec/EbMotionEstimationProcess.c]: The parameter is still set and used within the library

**Pull request merged**